cli/internal/fs/fs.go

```diff
@@ -78,6 +78,9 @@ func CopyFile(from *LstatCachedFile, to string) error {
 		if err := EnsureDir(to); err != nil {
 			return err
 		}
+		if _, err := os.Lstat(to); err == nil {
+			os.Remove(to)
```

Comment:
Can you handle the error here? Then I think this is ready.

Thanks!

Reply:
I don't know how to properly handle `os.Lstat` error, coz it's an allowed error state. Should I handle it then write some comments?